src/subarrayn.jl

```diff
@@ -0,0 +1,25 @@
+const unsafe_getindex = Base.unsafe_getindex
+
+function Base.isnull{T,N,P<:NullableArray,IV,LD}(V::SubArray{T,N,P,IV,LD}, I::Int...)
+    @Base._inline_meta
+    @boundscheck checkbounds(V, I...)
+    @inbounds r = V.parent.isnull[Base.reindex(V, V.indexes, Base.to_indexes(I...))...]
+    r
+end
+
+function Base.values{T,N,P<:NullableArray,IV,LD}(V::SubArray{T,N,P,IV,LD}, I::Int...)
+    @Base._inline_meta
+    @boundscheck checkbounds(V, I...)
+    @inbounds r = V.parent.values[Base.reindex(V, V.indexes, Base.to_indexes(I...))...]
+    r
+end
+
+@generated function anynull{T, N, U<:NullableArray}(S::SubArray{T, N, U})
```

Comment:
You didn't make any changes to this fubction, did you? Let's not make unnecessary duplications across versions here. I'd just put the version switch in the existing SubArray file.